lighthouse-core/lib/log.js

```diff
@@ -129,7 +129,40 @@ class Log {
     Log.events.issueStatus(title);
     return Log._logToStdErr(`${title}:verbose`, Array.from(arguments).slice(1));
   }
+
+  /**
+   * Add surrounding escape sequences to turn a string green when logged.
+   * @param {string} str
+   * @return {string}
+   */
+  static greenify(str) {
+    return `${Log.green}${str}${Log.reset}`;
+  }
+
+  /**
+   * Add surrounding escape sequences to turn a string red when logged.
+   * @param {string} str
+   * @return {string}
+   */
+  static redify(str) {
+    return `${Log.red}${str}${Log.reset}`;
+  }
+
 }
+
+Log.green = '\x1B[32m';
```

Comment:
I agree that would be the best, but definitions (of tick, yellow and other) are used outside this module for example in print.ts so I guess it has to go into export somehow.